bangazonapi/models/product.py

```diff
@@ -62,9 +62,16 @@ def average_rating(self):
         for rating in ratings:
             total_rating += rating.rating
 
-        avg = total_rating / len(ratings)
+        if total_rating > 0:
+
+            avg = total_rating / len(ratings)
+
```

Comment:
The ticket asks for a try..except block here rather than an if..else. See if you can apply a try..except to handle this ZeroDivisionError!

Reply:
I think this one is now resolved using try and except. Seems to be working in Postman.